### ISSUE TITLE
Archived completed projects

### ISSUE THREAD

**Issue by LamFTS** [kevboh/longform#250]
Dear Kevboh, I have used longform to manage my writing for a long period of time, therefore I have a long list of longform projects, is there anyway I can hidden or remove the projects that I have finished from the list? I would like to keep the `index` file in case of someday I might re-organize this project. May be limit the search location or add new property to yaml might be the potential solution.look forward to your reply :)

**Comment by kevboh** [comment]
Yep, it's something I want to do. In the meantime you could just edit the main `longform` yaml key to be, like, `longform-archived` or something.
